mbedtls/src/lib.rs

```diff
@@ -7,6 +7,12 @@
  * according to those terms. */
 
 #![deny(warnings)]
+// TODO: this is added to ensure `#![allow(ambiguous_glob_reexports)]` also works
+// in rust with old version, need to be removed after migration to rustc 1.70..,
+// See issue #277
+#![allow(unknown_lints)]
```

Comment:
After stabilization (i.e. during the 3.5.0), more `deny` directives should be added. IMO, a decent starting list is:
```rust
#![deny(
    unused,
    dead_code,
    missing_docs,
    missing_debug_implementations,
    rustdoc::broken_intra_doc_links,
    unreachable_pub,
    unused_allocation,
    unused_comparisons,
    unused_parens,
    while_true,
    trivial_casts,
    trivial_numeric_casts,
    unused_extern_crates,
    unused_import_braces,
    unused_qualifications,
    unused_results,
    clippy::all,
)]
#![warn(rust_2018_idioms)]
```